resources/win/atom.sh

```diff
@@ -12,15 +12,14 @@ else
     eval $(grep "^root" /etc/wsl.conf | sed -e "s/ //g")
     root="$(echo $root | sed 's|/|\\/|g')"
     ATOMCMD="$(echo $PWD | sed 's/\/mnt\/\([a-z]*\)\(.*\)/\1:\2/')/atom.cmd"
-    ATOMCMD="${ATOMCMD////\\}"
```

Comment:
> Not replace "//" with "" when building the ATOMCMD when detected in WSL

In [bash](https://linux.die.net/man/1/bash), this 👇 changes all the `/` in ATOMCMD into `\`:

https://github.com/atom/atom/blob/c5ebfbddd096a24bb99b832432a5dfcd67da4714/resources/win/atom.sh#L15